Gulpfile.js

```diff
@@ -43,17 +81,34 @@ gulp.task('flow', function() {
     .pipe(gulp.dest(paths.flowInclude));
 });
 
-gulp.task('browserify', ['lib'], function() {
-  return browserify(browserifyConfig)
-    .bundle()
-    .pipe(source('Flux.js'))
-    .pipe(gulp.dest(paths.dist))
+gulp.task('dist', ['lib'], function() {
+  var distOpts = {
+    debug: true,
+    output: 'Flux.js'
+  };
+  return gulp
+    .src(paths.entry)
+    .pipe(buildDist(distOpts))
+    .pipe(gulp.dest(paths.dist));
 });
 
-gulp.task('build', ['lib', 'flow', 'browserify']);
+gulp.task('dist:min', ['lib'], function() {
+  var distOpts = {
+    debug: false,
+    output: 'Flux.js'
+  };
+  return gulp
+    .src(paths.entry)
+    .pipe(buildDist(distOpts))
+    .pipe(gulp.dest(paths.dist));
+});
+
+gulp.task('build', ['lib', 'flow', 'dist']);
+
+gulp.task('build:min', ['lib', 'flow', 'dist:min']);
 
 gulp.task('publish', function(cb) {
-  runSequence('clean', 'build', cb);
+  runSequence('clean', 'build:min', cb);
```

Comment:
I think we actually want build and build:min to run so we get Flux and Flux.min. You also need lib to finish before the dist task. You might need run sequence for that too (not sure how well it dedupes when you already specify that dist needs lib, I just remember some pain in the past but if it's working then don't mess with it)